### PR TITLE
Added pythonanywhere.com to Web Hosting

### PR DESCRIPTION
**Pull request by Natnael123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -587,6 +587,7 @@ Table of Contents
   * [netlify.com](https://www.netlify.com/) — Builds, deploy and hosts static site or app, free for 100 GB data and 100 GB/month bandwidth
   * [pages.github.com](https://pages.github.com/) — Hosting static site directly from GitHub repository
   * [pantheon.io](https://pantheon.io/) — Drupal and WordPress hosting, automated DevOps and scalable infrastructure. Free for developers and agencies
+  * [pythonanywhere.com](https://www.pythonanywhere.com/) — Python hosting,develop and host your website or any other code directly from your browser without having to install software.
```
Comment (ripienaar) [inline README.md]: is already under PaaS :) maybe thats a bad category for it?

Comment (Natnael123) [reply README.md]: I guess hosting would be better.

Comment (ripienaar) [reply README.md]: feel free to move it then :)

**Comment by Natnael123** [comment]
smartmockups.com

**Comment by ripienaar** [comment]
thanks

**Pull request merged**